app/main/views/services.py

```diff
@@ -189,12 +189,7 @@ def get_service(service_id):
         Service.service_id == service_id
     ).first_or_404()
 
-    if service.framework.status == 'live':
-        return jsonify(services=service.serialize())
-    elif service.framework.status == 'expired':
-        abort(410)
-    else:
-        abort(404)
+    return jsonify(services=service.serialize())
```

Comment:
This only applies to the buyer frontend where [the change](https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/186) has already been merged. We have to remember that that must be deployed before this one.